default/templates/cmd/server/main.go

```diff
@@ -14,60 +13,41 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/spf13/viper"
 
-	"github.com/prometheus/client_golang/prometheus/promhttp"
-
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/stats"
-
-	"github.com/deciphernow/gm-fabric-go/metrics/grpcmetrics"
-	ms "github.com/deciphernow/gm-fabric-go/metrics/metricsserver"
-	pm "github.com/deciphernow/gm-fabric-go/metrics/prometheus"
 
 	"{{.ConfigPackage}}"
 	"{{.MethodsPackage}}"
 
 	// we don't use this directly, but need it in vendor for gateway grpc plugin
 	_ "github.com/golang/glog"
 	_ "github.com/grpc-ecosystem/grpc-gateway/runtime"
+	_ "github.com/ghodss/yaml"
 )
 
 func main() {
-	var tlsMetricsConf *tls.Config
 	var tlsServerConf *tls.Config
 	var err error
-	var zkCancels []zkCancelFunc
 
-	logger := zerolog.New(os.Stderr).With().Timestamp().Logger().
+	logger := zerolog.New(os.Stderr).
```

Comment:
Maybe change this to `stdout`? I'm okay with either though
```suggestion
	logger := zerolog.New(os.Stdout).
```